app/models/concerns/changed_at.rb

```diff
@@ -0,0 +1,11 @@
+module ChangedAt
+  extend ActiveSupport::Concern
+
+  class_methods do
```

Comment:
`class_methods`

Reply:
I'm pretty sure this is `class_methods &block`:

https://api.rubyonrails.org/classes/ActiveSupport/Concern.html#method-i-class_methods

Reply:
cool, just looked odd

Reply:
I like that you're starting to get a feel for what looks wrong or right in Ruby. :) 🏅